website/docs/compute/local-deployment/minikube.md

```diff
@@ -0,0 +1,225 @@
+
+Run the following `subo` command to deploy Compute to your cluster:
+
+```bash
+subo compute deploy core
+```
+
+### 7. Setup minikube tunneling
+
+Let’s tell minikube to forward requests to port 80 to our cluster!
+
+```bash
+minikube tunnel
+```
+
+### 8. Create an editor token
+
+In order to test our editor, we’re going to come up with a function name, and create a token so we can access it!
+
+This can only be done as [an API call to the control plane service](https://docs.suborbital.dev/compute/integrate-the-function-editor/code-editor) from within your cluster. Since we’re currently not running an app in our cluster, we’ll just make the call from within!
+
+First, we’ll need the name of our control plane pod:
+
+```bash
+kubectl get pod -n suborbital
+```
+
+Your output will look something like this:
+
+```bash
+NAME                                           READY   STATUS    RESTARTS   AGE
+scc-atmo-deployment-7bfb9d76c6-sv5dr           1/1     Running   0          27s
+scc-controlplane-deployment-5699f779f7-xmkhr   2/2     Running   0          27s
+```
+
+Let’s take that full name of our `scc-controlplane-deployment` pod and start a bash session inside it:
+
+```bash
+kubectl exec -n suborbital -it scc-controlplane-deployment-<REST OF POD CODENAME> -- bash
+```
+
+Would you look at that, we’re inside our cluster now!
+
+Let’s install `curl`:
+
+```bash
+apt update; apt install curl
+```
+
+With `curl` installed, we can now get our editor token for testing:
+
+```bash
+curl [http://local.suborbital.network:8081/api/v1/token/<IDENT>/default/](http://local.suborbital.network:8081/api/v1/token/com.acmeco.gr9fas97234b/default/httpget)<FUNCTION_NAME>
+```
+
+In which:
+
+- `IDENT`: Customer identity, for example: `com.example.12345`
+- `FUNCTION_NAME` : A name for your function
+
+This will give you a JSON response with a token. Let’s copy it!
+
+### 9. Try out the function editor
+
+The function editor is available through [building a specific URL](https://docs.suborbital.dev/compute/integrate-the-function-editor/code-editor). We can do that now that we have all the ingredients. In your browser, try opening up the following URL:
+
+```bash
+[https://editor.suborbital.network/?builder=https://<NGROK_DOMAIN>&token=<EDITOR_TOKEN>&ident=<IDENT>&fn=](https://editor.suborbital.network/?builder=https://4515-62-178-0-213.eu.ngrok.io&token=StIsWXsIAPJsjVlxcgItgvWS&ident=com.acmeco.gr9fas97234b&fn=ramono)<FUNCTION_NAME>&template=<LANGUAGE_TEMPLATE>
+```
+
+In which:
+
+- `NGROK_DOMAIN`: The domain generated by `ngrok` in step 3
+- `EDITOR_TOKEN`: The token generated by the control plane API in step 8
+- `IDENT`: Customer identity, for example: `com.example.12345`
+- `FUNCTION_NAME` : A name for your function
+- `LANGUAGE_TEMPLATE`: [A template to be prefilled](https://docs.suborbital.dev/compute/integrate-the-function-editor/code-editor#configuration) when opening the editor for a new function, defaulting to `AssemblyScript`.
```

Comment:
This link will change, too: https:/docs.suborbital.dev/compute/customizing-functions/code-editor#configuration